src/components/ec-inline-input-field/ec-inline-input-field.spec.js

```diff
@@ -1,11 +1,32 @@
 import { mount } from '@vue/test-utils';
 import clipboardCopy from 'clipboard-copy';
 import EcInlineInputField from './ec-inline-input-field.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
 
 jest.mock('clipboard-copy');
+jest.mock('../../directives/ec-tooltip', () => {
+  const MockedTooltipDirective = {
+    bind(el, { value }) {
+      // this can be undefined that's why we check it
+      if (value.content) {
+        el.setAttribute('mocked-tooltip-content', value.content);
+        el.setAttribute('mocked-tooltip-classes', value.classes);
+      }
+    },
+    update(el, { value }) {
+      if (value.content) {
+        el.setAttribute('mocked-tooltip-content', value.content);
```

Comment:
@stavros-tomas Those lines are repeated on lines 12 and 13, could we create a fn?

Reply:
Thank you @jmesao :)